src/golang/cmd/migrator/versions/000019_add_serialization_value_to_param_op/utils.go

```diff
@@ -0,0 +1,110 @@
+package _000019_add_serialization_value_to_param_op
+
+import (
+	"bytes"
+	"context"
+	"encoding/base64"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/aqueducthq/aqueduct/lib/collections/utils"
+	"github.com/aqueducthq/aqueduct/lib/database"
+	"github.com/google/uuid"
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	serilization_type_key   = "serialization_type"
+	value_key               = "val"
+	spec_field              = "spec"
+	pythonExecutorPackage   = "aqueduct_executor"
+	typeInferencePythonPath = "migrators.parameter_val_type_inference_000019.main"
+)
+
+type Operator struct {
+	Id     uuid.UUID `db:"id" json:"id"`
+	OpSpec Spec      `db:"spec" json:"spec"`
+}
+
+func getAllOperators(
+	ctx context.Context,
+	db database.Database,
+) ([]Operator, error) {
+	query := "SELECT id, spec FROM operator;"
+
+	var response []Operator
+	err := db.Query(ctx, &response, query)
+	return response, err
+}
+
+func updateParamOperatorWithNewSpec(
```

Comment:
Is this function idempotent? If some steps fail and we re-execute the migration script, will the already-migrated rows be affected? I think probably this is not idempotent now because we also tweaked the spec val?